src/bind.c

```diff
@@ -376,6 +378,7 @@ static SEXP vec_cbind(SEXP xs, SEXP ptype, SEXP size, struct name_repair_opts* n
                                               ptype,
                                               DF_FALLBACK_DEFAULT,
                                               S3_FALLBACK_false,
+                                              args_empty,
```

Comment:
So I guess `arg` could be added to `vec_rbind()`, `vec_cbind()`, and `vec_unchop()` too? At least in theory

Reply:
Yes in theory, though I think I'd prefer to introduce `df_cast_common()` first if needed.